install_piper.sh

```diff
@@ -1,78 +1,94 @@
 #!/bin/bash
+#!/bin/bash
 
-# Exit immediately if a command exits with a non-zero status
-set -e
-
-# Determine system architecture
-ARCH=$(uname -m)
+function main() {
+     
+    # Check for help flag
+    if [[ "$@" == "--help" || "$@" == "-h" ]]; then
+      echo "Usage: source install_piper.sh [download] [model]"
+      echo "   download: Set to true to download. Default is true."
```

Comment:
we should keep default as false, don't want users disk to populate unncessarily. Hence the prompt later to ask permission explicitly. Kinda like what other installers do

Reply:
changed